open-api/rest-catalog-open-api.yaml

```diff
@@ -770,6 +770,40 @@ paths:
         5XX:
           $ref: '#/components/responses/ServerErrorResponse'
 
+  /v1/{prefix}/metrics/scan:
+    parameters:
+      - $ref: '#/components/parameters/prefix'
+
+    post:
+      tags:
+        - Catalog API
+      summary: Send a scan report to this endpoint to be processed by the backend
+      description:
+        Send a scan report to this endpoint to be processed by the backend
+      operationId: sendScanReport
+      requestBody:
+        description: The request containing the scan report to be sent
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/ScanReportRequest'
+        required: true
+      responses:
+        200:
+          description: OK
+        400:
+          $ref: '#/components/responses/BadRequestErrorResponse'
+        401:
+          $ref: '#/components/responses/UnauthorizedResponse'
+        403:
+          $ref: '#/components/responses/ForbiddenResponse'
```

Comment:
Since this is informational, I think it would make sense to simply return a 200 instead of 401 or 403. I guess it's up to the service though. I doubt we'd want to send errors back to the client on this route.